src/datetime/mod.rs

```diff
@@ -831,6 +826,11 @@ impl DateTime<Utc> {
     pub fn parse_from_str(s: &str, fmt: &str) -> ParseResult<DateTime<Utc>> {
         DateTime::<FixedOffset>::parse_from_str(s, fmt).map(|result| result.into())
     }
+
+    /// The minimum possible `DateTime<Utc>`.
+    pub const MIN_UTC: Self = DateTime { datetime: NaiveDateTime::MIN, offset: Utc };
```

Comment:
Should we lose the `_UTC` suffix now that it's sort of implied by the `<Utc>` type param?

Reply:
Previously you would have to write `let dt = DateTime<Utc>::MIN_UTC`, and with this change it becomes `let dt = DateTime::MIN_UTC`.
With the change I think keeping `_UTC` it makes code more readable.